mlflow/ml-package-versions.yml

```diff
@@ -343,3 +343,19 @@ mleap:
         exit 1
       fi
       pytest tests/mleap/test_mleap_model_export.py --large
+
+pmdarima:
+  package_info:
+    pip_release: "pmdarima"
+    install_dev: |
+      if [ ! -d "$CACHE_DIR" ] || [ -z $(find $CACHE_DIR -type f -name "pmdarima-*.whl") ]; then
+        pip wheel --no-deps --wheel-dir $CACHE_DIR git+https://github.com/alkaline-ml/pmdarima.git
+      fi
+      pip install $CACHE_DIR/pmdarima-*.whl
+
+  models:
+    minimum: "1.8.4"
```

Comment:
We seem to be compatible with pmdarima < 1.8.4 based on your warning logic for the case where exogenous regressors are specified with `pmdarima < 1.8.0`. Can we drop the minimum version down to 1.8.0, providing support for all versions of pmdarima released within the last year (+ December 2020)?

Reply:
+1

Reply:
changed and upped the cross version test to 1.8.5 (released 2 days ago)